test/Reflection/conformance_descriptors.swift

```diff
@@ -9,7 +9,7 @@
 // RUN: %empty-directory(%t)
 
 // RUN: %target-build-swift %S/Inputs/ConcreteTypes.swift %S/Inputs/GenericTypes.swift %S/Inputs/Protocols.swift %S/Inputs/Extensions.swift %S/Inputs/Closures.swift %S/Inputs/Conformances.swift -parse-as-library -emit-module -emit-library -module-name ConformanceCheck -o %t/Conformances
-// RUN: %target-swift-reflection-dump %t/Conformances %platform-module-dir/%target-library-name(swiftCore) | %FileCheck %s
+// RUN: %target-swift-reflection-dump %t/Conformances %platform-dylib-dir/%target-library-name(swiftCore) | %FileCheck %s
```

Comment:
Now that the `platform-dylib-dir` is different from the `platform-module-dir`, this is the right path. Most tests use this `platform-dylib-dir` formulation, except for [two more](https://github.com/apple/swift/commit/52353f8a89f8#diff-9e12fb53e0749feeebf2fb103ce9a5253f7d807d384ff0f9d69c5be2ac40ea28R17) mac-only [tests added in recent years](https://github.com/apple/swift/commit/c3df37cff338#diff-40d62e3d1a7c53f804d6dbd63d6db03febafb6b3fe619d3862ee83c3165c0585R8).

@artemcm or @al45tair, let me know if I should change those two also.